### ISSUE TITLE
How to validate the Controlnet?

### ISSUE THREAD

**Issue by jiajiao272727** [lllyasviel/ControlNet#745]
Hi, I'm a code beginner. I've learned how to train a model, and I'd like to learn how to validate it and get a dynamic validation loss.

I tried adding `check_val_every_n_epoch=1` inside `trainer = pl.Trainer()` in tutorial_train_sd21.py, but the output is still only the training loss.

Do I need to modify the file that creates the loss function to distinguish between the train_step and validation_step fields? How can I do this?
Thank you very much!
